src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/NodeBuilder.cs

```diff
@@ -184,7 +184,7 @@ public CoreNode CreateCustomNode(Action<IFullNodeBuilder> callback, Network netw
             string dataDir = configParameters["datadir"];
 
             configParameters.ToList().ForEach(p => this.ConfigParameters[p.Key] = p.Value);
-            return CreateNode(new CustomNodeRunner(dataDir, callback, network, protocolVersion, configParameters, agent, minProtocolVersion), configFileName);
+            return CreateNode(new CustomNodeRunner(dataDir, callback, network, protocolVersion, configParameters, agent, minProtocolVersion), configFileName, configParameters: configParameters);
```

Comment:
Why are we passing `configParameters` twice?

Reply:
Original code passes configParameters into the CustomRunner constructor but this member is private and so not accessible to the Core node for the purpose of setting its own ConfigParameters member.